cleanenv.go

```diff
@@ -286,6 +286,10 @@ func readEnvVars(cfg interface{}, update bool) error {
 			}
 		}
 
+		if rawValue == nil && meta.fieldValue.IsZero() {
```

Comment:
The IsZero() call requires Go 1.13 or better.

The point of this change is to
* use env value if supplied, or
* use value in config if supplied, or
* use default value if supplied.